CONTRIBUTING.md

```diff
@@ -123,4 +124,4 @@ repository, we have a [community membership guide](./docs/governance/community_m
   - Info - should be used to document state changes in the application or some entity within the application. These logs provide the skeleton of what has happened. Examples: system started/stopped, remote API calls, a new user has been created/updated, etc.
   - Debug - diagnostic information goes here and everything that can help to troubleshoot an application. Examples: any values in business logic, detailed information about the data flow.
 
-  More details can be found here: https://reflectoring.io/logging-levels/
\ No newline at end of file
+  More details can be found here: https://reflectoring.io/logging-levels/
```

Comment:
This link is not accessible:
 https://reflectoring.io/logging-levels/
ERR_HTTP2_PROTOCOL_ERROR

Reply:
It could be a copy of this blog post:
https://medium.com/@tom.hombergs/tip-use-logging-levels-consistently-913b7b8e9782

Reply:
The link works for me, but comparing to the medium article they look identical.

Reply:
Pushed a commit with the medium link.